scripts/start-deployPufferfish

```diff
@@ -8,7 +8,7 @@ isDebugging && set -x
 
 IFS=$'\n\t'
 
-if [ "${VANILLA_VERSION%.*}" != "1.18" ] && [ "${VANILLA_VERSION%.*}" != "1.17" ]; then
+if [[ "${VANILLA_VERSION}" != "1.18" ]] && [[ "${VANILLA_VERSION}" != "1.17" ]]; then
```

Comment:
It looks like the retrieval on line 18 also needs a similar tweak.

Reply:
Changed!